include/roboptim/core/visualization/matplotlib.hh

```diff
@@ -105,6 +111,9 @@ namespace roboptim
 
       /// \brief Whether to export the header (imports etc.).
       bool withHeader_;
+
+      /// \brief Whether to expect multiple plots or not.
+      unsigned int multiplot_;
```

Comment:
I think a `std::pair<unsigned int, unsigned int>` would be better to specific the layout, rather than forcing the plot on a single column.